test/cri-containerd/container_test.go

```diff
@@ -874,3 +874,73 @@ func Test_RunContainer_ExecUser_Root_LCOW(t *testing.T) {
 		t.Fatalf("expected user for exec to be 'root', got %q", string(r.Stdout))
 	}
 }
+
+// creates two sandboxes and starts two containers within them. The first sandbox then starts a NFS server and
+// the second container tries to mount that sandbox.
+func Test_Container_NFSMount_LCOW(t *testing.T) {
+	requireFeatures(t, featureLCOW)
+
+	pullRequiredLCOWImages(t, []string{imageLcowK8sPause, imageLcowAlpine, imageLcowUbuntu})
+
+	client := newTestRuntimeClient(t)
+	ctx, cancel := context.WithCancel(context.Background())
+	defer cancel()
+
+	// start a privileged pod & container. container must be privileged in order to be able to mount a NFS
+	// share.
+	sandboxRequest := getRunPodSandboxRequest(t, lcowRuntimeHandler)
+	sandboxRequest.Config.Linux = &runtime.LinuxPodSandboxConfig{
+		SecurityContext: &runtime.LinuxSandboxSecurityContext{
+			Privileged: true,
+		},
+	}
+	podID := runPodSandbox(t, client, ctx, sandboxRequest)
+	defer removePodSandbox(t, client, ctx, podID)
+	defer stopPodSandbox(t, client, ctx, podID)
+
+	requestTemplate := getCreateContainerRequest(
+		podID,
+		t.Name()+"-container",
+		imageLcowUbuntu,
+		[]string{"bash", "-c", "while true; do echo 'hello'; sleep 1; done"},
+		sandboxRequest.Config,
+	)
+	requestTemplate.Config.Linux = &runtime.LinuxContainerConfig{
+		SecurityContext: &runtime.LinuxContainerSecurityContext{
+			Privileged: true,
+		},
+	}
+	containerID := createContainer(t, client, ctx, requestTemplate)
+	defer removeContainer(t, client, ctx, containerID)
+	startContainer(t, client, ctx, containerID)
+	defer stopContainer(t, client, ctx, containerID)
+
+	execHelper := func(ctrID string, cmd []string) (string, string, int) {
+		stdout, stderr, errcode := execContainer(t, client, ctx, ctrID, cmd)
+		if errcode != 0 {
+			t.Logf("stdout: %s \n\n stderr: %s\n\n", stdout, stderr)
```

Comment:
can you add a `t.Helper()` call before the `t.Logf` call?

Reply:
Do we want the `t.Helper()` to be in the errcode block of code or outside of it?

Reply:
Shouldn't matter I think. Either works.